tests/integration/sugar/test_account.py

```diff
@@ -0,0 +1,45 @@
+from unittest import TestCase
+
+from tests.integration.it_utils import JSON_RPC_CLIENT
+from tests.integration.reusable_values import WALLET
+from xrpl.account import does_account_exist, get_account_transactions, get_balance
+from xrpl.wallet import Wallet, generate_faucet_wallet
+
+NEW_WALLET = generate_faucet_wallet(JSON_RPC_CLIENT)
```

Comment:
maybe wait to merge this after mine - i wanna have two faucets in the shared values so we can make cases like this faster

Reply:
my open PR has a `DESTINATION` wallet that is also generated from faucet